### PR TITLE
fix(helloworld): fix doc content to 1d array

### PR DESCRIPTION
**Pull request by hanxiao**

### CODE REVIEW

File: jina/helloworld/fashion/my_executors.py [hunk 1]
```diff
@@ -64,9 +64,10 @@ def encode(self, docs: 'DocumentArray', **kwargs):
         # content.shape=(request_size, 28, 28, 3)
         content = content[:, :, :, 0].reshape(-1, 784)
         # content.shape=(request_size, 784)
-        embeds = (content.reshape([-1, 784]) / 255) @ self.oth_mat
-        for doc, embed in zip(docs, embeds):
+        embeds = (content / 255) @ self.oth_mat
+        for doc, embed, cont in zip(docs, embeds, content):
             doc.embedding = embed
+            doc.content = cont
```
Comment (alaeddine-13) [inline jina/helloworld/fashion/my_executors.py:70]: why is this needed ?
I tested without it and demo.html looks fine

**Review by davidbp — APPROVED** [review]
Working as expected, LGTM
